paper.tex

```diff
@@ -257,30 +257,30 @@ \section{Vision}%
 Thus, it is likely that no two RSE units will be, or should be, alike.
 However, these nine modules together with assumed weights are part of a simple model of an RSE group which provides both a quick overview of an individual group as well as a way to compare groups.
 The nine modules are described below.
-\todo{FL:\@ Introduce the hub-spokes terminology.}
 
 \subsection{Module 1: Foster a local Network of RSEs}%
 \label{sec:network}
 
-One of the core responsibilities of an RSE unit is to act as a coordinator of RSE activities within the institution.
-At virtually every academic institution there are employees that assume at least part-time the role of an RSE.
+One of the core responsibilities of an RSE unit is to act as a coordinator of RSE activities across the institution.
+Research software is an integral part of modern research. 
+There are, therefore, researchers in every institution that at least sometimes perform the tasks of an RSE.
```

Comment:
```suggestion
There are, therefore, researchers in every institution that at least sometimes perform the tasks of an RSE. We call these RSEs embedded RSEs.
```

Reply:
Maybe we don't need an entire new sentence and implicitly define what embedded RSEs are. like with dd8c2bb. I will still add something to the intro

Reply:
I orefer it this way.